### ISSUE TITLE
group deletion should be disallowed when containers are belonging to a group

### ISSUE THREAD

**Issue by jainvipin** [contiv/netplugin#433]
Say I create a network and policy, and start a container as follows:
```
netctl net create -s 10.1.1.0/24 contiv-net
netctl policy create allow-all
netctl group create contiv-net happy-group -policy=allow-all
docker run -itd --name=happy-container1 --net=happy-group alpine /bin/sh

netctl group rm happy-group
```
In the above sequence, should it disallow removing a group when container `happy-container1` was belonging to the group, consistent with the fact that I am unable to delete a network when containers are belonging to it. On other hand `docker network inspect happy-group` is showing that all is ok, and I am also able to recreate the group again.

**Comment by gaurav-dalvi** [comment]
Problem is not see when you do following.
netctl net create -s 10.1.1.0/24 contiv-net
docker run -itd --name=happy-container1 --net=contiv-net alpine /bin/sh
netctl net rm contiv-net.

INFO[0000] Deleting network default:contiv-net                  
ERRO[0000] Error: Network has active endpoints [github.com/contiv/netplugin/netmaster/master.DeleteNetworkID network.go 415]

It occurs only when you have EPG in picture.

**Comment by gaurav-dalvi** [comment]
@DivyaVavili : Looks like this is fixed now. Could you please close this issue then ?

**Comment by DivyaVavili** [comment]
Fixed by PR #484
